src/components/ShowContextMenuContext.ts

```diff
@@ -55,7 +57,7 @@ function showContextMenuForReport(
         action?.reportActionID,
         ReportUtils.getOriginalReportID(reportID, action),
         undefined,
-        checkIfContextMenuActive,
+        undefined,
```

Comment:
Is that change correct? Seems like we removed existing code? (if we don't need it anymore its probably good to cleanup the usage of `checkIfContextMenuActive`)